commonFuctions/index.tsx

```diff
@@ -382,3 +383,77 @@ export const getPdfThumbnail = async (selectedFile: any) => {
   }
   return arr;
 };
+
+export function splitWordsWithSpace(input: string) {
+  const words = input.split(' ');
+
+  for (let i = 0; i < words.length; i++) {
+    if (words[i + 1] === '') {
+      words[i] += ' ';
+    }
+  }
+
+  return words;
+}
+
+export function detectMentions(input: string) {
+  const mentionRegex = /@(\w+)/g;
+  const matches = [];
+  let match;
+
+  while ((match = mentionRegex.exec(input)) !== null) {
+    const startIndex = match.index;
+    const endIndex = mentionRegex.lastIndex;
+    const nextChar = input.charAt(endIndex);
+
+    if (
+      nextChar !== ' ' &&
+      nextChar !== '@' &&
+      match[1] !== 'everyone' &&
+      match[1] !== 'participants'
+    ) {
+      matches.push(match[1]);
+    }
+  }
+
+  if (input.endsWith(' @') || input === '@' || input.endsWith('\n@')) {
+    matches.push('');
+  }
```

Comment:
why these checks? please mention comments? and what if in future, we create more group tags, then you will need to create a release? so remove these hard checks on "everyone" and "participants"